test/proxies_test.js

```diff
@@ -199,7 +200,7 @@ describe('Automerge proxy API', () => {
           assert.strictEqual(doc.list.every(val => val > 0), true)
           assert.strictEqual(doc.list.every(val => val > 2), false)
           assert.strictEqual(doc.list.every((val, index) => index < 3), true)
-          doc.list.every(function () { assert.strictEqual(this.hello, 'world') }, {hello: 'world'})
+          doc.list.every(function () { assert.strictEqual(this.hello, 'world') }, {hello: 'world'}) // eslint-disable-line
```

Comment:
I really don't understand what the intent is with these calls. Is `this` changing on each iteration?

Reply:
The intent is to check that when the callback function is called, `this` is set to the second argument passed to `every` (to match the behaviour of [Array.every](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/every)). I added a comment to explain.